tests/Test.MSAL.NET.Unit.PublicApi/ReadMe.txt

```diff
@@ -1,7 +0,0 @@
-﻿Unit tests for the public API only
-----------------------------------
-
-It is important this project is *not* able to see the internals of the product assemblies,
-which is why these tests are in a separate project (the product internals are visible to
-"Test.MSAL.Net.Unit").
```

Comment:
Public API can also be tested in a single project. It is an additional project overhead that I's rather not maintain. You can create separate test files if needed (i don't think its necessary).